src/scratchtocatrobat/converter/test_converter.py

```diff
@@ -2395,14 +2394,46 @@ def _test_project(self, project_name):
 
         scratch_project = scratch.Project(scratch_project_dir, name=project_name,
                                           project_id=common_testing.PROJECT_DUMMY_ID)
+        return scratch_project
 
+    def _test_project(self, project_name):
+        scratch_project = self._load_test_scratch_project(project_name)
         context = converter.Context()
         converted_project = converter.converted(scratch_project, None, context)
         catrobat_zip_file_name = converted_project.save_as_catrobat_package_to(self._testresult_folder_path)
         self.assertValidCatrobatProgramPackageAndUnpackIf(catrobat_zip_file_name, project_name,
                                                           unused_scratch_resources=scratch_project.unused_resource_names)
         return converted_project.catrobat_program
 
+    # Checks if the visible global or local variables in the scratch program are converted into show test bricks in the converted project
+    def test_can_convert_visible_variables(self):
+        scratch_project = self._load_test_scratch_project("visible_variables")
+        visibility_map = scratch_project._var_to_visibility_map
+        catrobat_program = self._test_project("visible_variables")
+        scene = catrobat_program.getDefaultScene()
+        sprite_dict = dict()
+        sprite_list = scene.getSpriteList()
+        for sprite in sprite_list:
+            sprite_name = sprite.getName()
+            # TODO: rewrite when there is a general name for the background sprite in the code
+            if sprite_name == "Hintergrund":
+                sprite_name = "Stage"
+            sprite_dict[sprite_name] = sprite
+        for var in visibility_map:
+            if visibility_map.get(var)[0]:
+                target_sprite = visibility_map.get(var)[1]
```

Comment:
Please use a more descriptive name like `sprite_name` instead of `target_sprite` in order to conform to our clean code guidelines.
Otherwise the reader of your code has to think about the meaning of the variable and might get confused.
However, I know that this particular name originates from the JSON code of the Scratch project and you just wanted to use the same name for the variable.